ajax/libs/redux-logger/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "redux-logger",
+  "filename": "redux-logger.js",
+  "version": "3.0.6",
+  "description": "Logger for Redux",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/theaqua/redux-logger.git"
+  },
+  "keywords": [
+    "redux",
+    "logger",
+    "redux-logger",
+    "middleware"
+  ],
+  "author": {
+    "name": "Eugene Rodionov",
+    "url": "https://github.com/theaqua"
```

Comment:
Let's stick to the author format in original [package.json](https://unpkg.com/redux-logger@3.0.6/package.json)?

`"author": "Eugene Rodionov (https://github.com/theaqua)"`